### PR TITLE
Styling v2

### PR DESCRIPTION
**Pull request by gvonkoss**
@TheoLeanse @andygnewman

### CODE REVIEW

**Comment by andygnewman** [comment]
Should add demos/public to your .gitignore

**Comment by gvonkoss** [comment]
it's in there ...?

**Comment by TheoLeanse** [comment]
erp but we must keep `demos/index.html` and `demos/load-demos.js`, only burning the `demos/public` stuff

**Comment by gvonkoss** [comment]
erp derp

File: demos/index.html [hunk 1]
```diff
@@ -5,11 +5,11 @@
 		<meta charset="utf-8" />
 		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
 		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-typography@^4.3.2" />
+		<link rel="stylesheet" href="https://build.origami.ft.com/v2/bundles/css?modules=o-typography@^4.3.2,o-icons@^5.2.0,o-colors@^3.5.4" />
```
Comment (TheoLeanse) [inline demos/index.html]: I'm just thinking out loud but maybe we don't need to use the build service for our demos when we now have our build step and our bower components and our sass?

**Review by TheoLeanse — APPROVED** [review]

File: demos/index.html (hunk 1 above)
Comment (gvonkoss) [reply demos/index.html]: Dunno. Let's find out!

**Pull request merged**